.coveragerc

```diff
@@ -1,15 +0,0 @@
-# Useful links in setting up:
-# https://github.com/codecov/example-python
-# https://coverage.readthedocs.io/en/latest/config.html
-# also, I looked at xgcm/xmitgcm as "templates"
-[run]
-branch = True
-
-[report]
-exclude_lines =
-    raise NotImplementedError
-    if __name__ == .__main__.:
-    if not less_output:
-ignore_errors = True 
-omit = ecco_v4_py/test/*
-       ecco_v4_py/__init__.py
```

Comment:
Is there a reason coveragerc got deleted?

Reply:
To answer the more technical part, I think no: the pyresample routines expect a 1D array of "swath" data but allow for an extra dimension ("channels" in the pyresample documentation) along which no interpolation occurs, conventionally the final dimension. The trick here is just to take any dimensions along which no interpolation is desired (such as model level or time) and stack them into one final dimension to be treated as "channels". They get unstacked and put back in their original location after the interpolation. 

Regarding `.coveragerc`, I think it appeared after I ran the tests so I assumed this was some sort of test-specific output data without much thought. Thinking about it the `rc` would suggest otherwise. My mistake.